### PR TITLE
[HeaderStackView] Update swift example to not depend on obj-c example

### LINKED ISSUE #4781: {Swift} Several examples depend on Objective-C example code
* ShadowElevationsTypicalUseExample.swift depends on ShadowElevationsPointsLabel.h
* NavigationBarTypicalUseExample.swift depends on NavigationBarTypicalUseExampleSupplemental.{h,m}
* HeaderStackViewTypicalUseSwiftExample.swift depends on HeaderStackViewTypicalUseSupplemental.{h,m}

### PR DESCRIPTION
**Pull request by codeman7**
Partially closes #4781

### CODE REVIEW

**Comment by romoore** [comment]
Xcode 8.3.3 failure:

```
[x] /Volumes/BuildData/tmpfs/src/github/repo/components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift:145:37: 'count' is unavailable: there is no universally good answer, see the documentation comment for discussion

    let endLength = instructionText.count - 11
                                    ^~~~~
```

File: components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift [hunk 1]
```diff
@@ -45,4 +51,118 @@ open class HeaderStackViewTypicalUseSwiftExample: HeaderStackViewTypicalUse {
...
+      NSFontAttributeName : UIFont.preferredFont(forTextStyle: .subheadline),
+      NSForegroundColorAttributeName : MDCPalette.grey.tint600.withAlphaComponent(0.87),
+      NSParagraphStyleAttributeName : style
+    ]
+
+    let instructionText = "SWIPE RIGHT\n\n\nfrom left edge to go back\n\n\n\n\n"
+    let instructionsAttributedString = NSMutableAttributedString(string: instructionText)
+    instructionsAttributedString.addAttributes(instructionsDictionary1,
+                                               range: NSMakeRange(0, 11))
+    let endLength = instructionText.count - 11
```
Comment (romoore) [inline components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift]: Perhaps character.count?

https://stackoverflow.com/questions/24037711/get-the-length-of-a-string

File: components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift [hunk 2]
```diff
@@ -45,4 +51,118 @@ open class HeaderStackViewTypicalUseSwiftExample: HeaderStackViewTypicalUse {
...
+    style.lineBreakMode = .byWordWrapping
+    let instructionsDictionary1 = [
+      NSFontAttributeName : UIFont.preferredFont(forTextStyle: .headline),
+      NSForegroundColorAttributeName : MDCPalette.grey.tint600.withAlphaComponent(0.87),
+      NSParagraphStyleAttributeName : style
+    ]
+
+    let instructionsDictionary2 = [
+      NSFontAttributeName : UIFont.preferredFont(forTextStyle: .subheadline),
+      NSForegroundColorAttributeName : MDCPalette.grey.tint600.withAlphaComponent(0.87),
```
Comment (romoore) [inline components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift:137]: We have a color scheme, was it unused in the previous example code?

**Review by romoore — APPROVED** [review]

File: components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift (hunk 1 above)
Comment (codeman7) [reply components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift]: Yeah I missed that one.

File: components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift (hunk 2 above)
Comment (codeman7) [reply components/HeaderStackView/examples/HeaderStackViewTypicalUse.swift:137]: Yeah it was unused.

**Pull request merged**